src/xcode/ENA/ENA/Source/Scenes/Home/HomeCoordinator.swift

```diff
@@ -493,6 +493,29 @@ class HomeCoordinator: RequiresAppDependencies {
 			.store(in: &subscriptions)
 	}
 
+	private func setupRiskChangeHomeBadgeCount() {
+		guard let homeState = homeState else {
+			Log.error("HomeState missing can not setup badge count")
+			return
+		}
+
+		homeState.$riskState
+			.receive(on: DispatchQueue.OCombine(.main))
+			.sink { [weak self] riskState in
+				// check if risk level raised and if home screen tab is not selected
+				guard case let .risk(risk) = riskState,
+					  risk.riskLevelHasChanged,
+					  risk.level == .high,
+					  self?.rootViewController.tabBarController?.selectedViewController != self?.rootViewController
+				else {
+					Log.info("wrong risk level or home screen tab is active - skipped to set tab bar badge")
+					return
+				}
+					self?.rootViewController.tabBarItem.badgeValue = "1"
```

Comment:
I think we should somehow combine this with the value `func setupHomeBadgeCount()` above, otherwise these will just overwrite each other and not count up correctly.

Reply:
good point, I'll convert it back to a draft and work on a solution.